consensus/types/src/consts.rs

```diff
@@ -32,9 +32,9 @@ pub mod deneb {
             "52435875175126190479447740508185965837690552500527637822603658699938581184513"
```

Comment:
just discussed with @michaelsproul and might be worth moving this to a const as well